Encoding issues in IntelliJ

# About

Fix encoding issues:
```
[...]\features\basic\SuggestionsUpDownVoter.java:27: error: unmappable character (0x8D) for encoding windows-1252
    private static final Emoji FALLBACK_UP_VOTE = Emoji.fromUnicode("�?");
```

Add to `build.gradle`:
```
    compileJava {
        options.encoding = "UTF-8"
    }

    compileTestJava {
        options.encoding = "UTF-8"
    }
```

--- a/build.gradle
+++ b/build.gradle
@@ -86,4 +86,12 @@ subprojects {
     test {
         useJUnitPlatform()
     }
+
+    compileJava {
+        options.encoding = "UTF-8"
+    }
+
+    compileTestJava {
+        options.encoding = "UTF-8"
+    }
 }

